packages/appsync-modelgen-plugin/tsconfig.json

```diff
@@ -1,13 +1,18 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "rootDir": "src",
-    "outDir": "lib"
+    "rootDir": ".",
```

Comment:
These changes, along with the addition of `src/tsconfig.json` are necessary for importing the version from `package.json`.